draft/draft-ietf-rtgwg-qos-model.xml

```diff
@@ -234,26 +234,49 @@
     </section>
     
     <section title="QoS Model Design">
+      <figure>
+        <name>QoS Yang Modules Layout</name>
+        <artwork>
+         -------------   ------------   ------------   ------------
+        |             | |            | |            | |            |
+        | Diffserv    | | Queue      | | Scheduling | | QoS        |
+        | Policy      | | Policy     | | Policy     | | Oper       |
+        |             | |            | |            | |            |
+         -------------   ------------   ------------   ------------
+         -------------   ------------------------------------------
+        |             | |                                          |
+        | QoS Action  | |  Traffic Policy                          |
+        |             | | (Base Policy, Classifier, Target)        |
+        |             | |                                          |
+         -------------   ------------------------------------------
+        </artwork>
+      </figure>
+
+      <t> QoS Action and Traffic Policy are base Modules which are 
+      used by Diffserv, Queue and Scheduling Policy Modules as well
+      as by QoS Oper Module to derive respective module functionality.
+      </t>
+
```

Comment:
How about this as text under the diagram.

"This above diagram depicts the design of the QoS model. It uses two models as base models at the bottom, which are augmented by the remaining models on the top to define different aspects of QoS. This structure allows other models such as Policy Based Routing (PBR) <cite any RFC here> to augment the base model and define PBR."

What we then need to add is the linkage between the diagram and names in the diagram above and some of the following text.

There is no description of the Scheduling Policy module in the draft. Same for Queue Policy module.

Reply:
Works for me.